base/abstractarray.jl

```diff
@@ -2299,8 +2302,9 @@ function _typed_hvncat(::Type{T}, shape::Tuple{Vararg{Tuple, N}}, row_first::Boo
     shapepos = ones(Int, nd)
 
     for i ∈ eachindex(as)
+        length(as[i]) > 0 || ArgumentError("argument $i has no elements") |> throw
```

Comment:
This is the smallest of nits, but since you use the functional form above let's at least be consistent within one patch :)

```suggestion
        length(as[i]) > 0 || throw(ArgumentError("argument $i has no elements"))
```

Reply:
In the other PR I shifted everything else to `throw(...)`, so thanks :-)